_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Istio Multicluster
+overview: Install Istio with multicluster support.
+
+order: 65
+
+layout: docs
+type: markdown
+---
+
+{% include home.html %}
+
+Instructions for the installation of Istio multicluster.
+
+## Prerequisites
+
+* Individual cluster Pod CIDR ranges and service CIDR ranges must be unique
+across the multicluster environment and may not overlap.
+
+* All nodes' pod CIDR in every cluster must be routable to every other nodes'
+pod CIDR.
+
+* All Kubernetes control planes must be routeable to each other.
+
+* Helm **2.7.2 or newer**.  The use of Tiller is optional.
+
+* If using Tiller, the ability to modify RBAC rules required to install Helm
+or alternatively Helm should already installed.
+
+* Kubernetes **1.7.3 or newer**.
+
+* Istio secret `istio.default` replicated to all remote Kubernetes clusters
+from the Istio control plane per application namespace.
+
+* The local cluster and all remote Kubernetes cluster credentials (typically
+stored in `$HOME/.kube/config`) copied to
+`$HOME/multicluster/${UNIQUE_IDENTIFIER}.kube.conf`.
+
+* A [deployed Istio control plane]({{home}}/docs/setup/kubernetes/quick-start.html)
+on **one** Kubernetes cluster.  Note that Iatio must be [deployed with Helm]({{home}}/docs/setup/kubernetes/helm-install.html) using the `--set global.multicluster.enabled=true` flag.  This is currently mandatory.
```

Comment:
typo Iatio -> Istio

Reply:
Not sure how I missed that, pushing an update now :)